dev-client/src/screens/HomeScreen/components/TemporaryLocationCallout.tsx

```diff
@@ -80,32 +86,30 @@ export const TemporaryLocationCallout = ({
       buttons={<CloseButton onPress={closeCallout} />}
       isPopover={true}>
       <Column mt="12px" space="12px">
-        {!isSoilIdReady && <ActivityIndicator size="small" />}
-        {topSoilMatch && (
-          <>
-            <CalloutDetail
-              label={t('site.soil_id_prediction')}
-              value={topSoilMatch.soilInfo.soilSeries.name}
-            />
-            <Divider />
-            <CalloutDetail
-              label={t('site.ecological_site_prediction')}
-              value={
-                topSoilMatch.soilInfo.ecologicalSite?.name ??
-                t('site.soil_id.soil_info.no_matches')
-              }
-            />
-            <Divider />
-          </>
-        )}
-        {elevation ? (
+        <>
+          <CalloutDetail
+            label={t('site.soil_id_prediction')}
+            value={soilMatchDisplay}
+          />
+          <Divider />
           <CalloutDetail
-            label={t('site.elevation_label')}
-            value={renderElevation(t, elevation)}
+            label={t('site.ecological_site_prediction')}
+            value={ecologicalSiteDisplay}
           />
-        ) : (
-          <ActivityIndicator size="small" />
-        )}
+          <Divider />
+        </>
+        <CalloutDetail
+          label={t('site.elevation_label')}
+          value={
+            elevation ? (
```

Comment:
question: will this handle the elevation data timing out/returning an error?

Reply:
No, this is purely for soil ID. I expect that to be considered elevation-related work. Is that something we need to track (either for Capri or later)?
Also, just FYI so far we don't handle anything timing out.

Reply:
I also created https://github.com/techmatters/terraso-product/issues/902 for us to consider error behaviors in general.